Source/WebKit/UIProcess/Launcher/cocoa/ProcessLauncherCocoa.mm

```diff
@@ -148,19 +148,26 @@ static void launchWithExtensionKit(ProcessLauncher& processLauncher, ProcessLaun
 
 void ProcessLauncher::launchProcess()
 {
+    //WTF::sleep(30_s);
```

Comment:
Nit - it seems like this was just for debugging?
```suggestion
```

Reply:
(It also wasn't clear if the `fprintf` and `fflush` below were intended — if so, it might be to make `"Returning early"` a bit more descriptive)

Reply:
Ah, good points! I'll remove these in the next patch.

Thanks for reviewing!